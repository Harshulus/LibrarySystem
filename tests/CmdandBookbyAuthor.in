meh
add collection SampleCollection.csv
author Harper Lee
author jake
