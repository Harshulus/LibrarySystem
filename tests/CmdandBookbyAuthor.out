user: Give the correct command.

user: 17 books successfully added.

user: To Kill a Mockingbird (Harper Lee)

user: No books by jake.

